autoform-api.js

```diff
@@ -309,6 +309,7 @@ AutoForm.getTemplateName = function autoFormGetTemplateName (
   }
 
   // Found nothing. Return undefined
+  console.warn('AutoForm: no theme found, please consult the README.')
```

Comment:
In the future we could use the `logging` package to do something like `Log.debug`, but since we need this warning before this comes anywhere near production this is adequate.